app/web/package.json

```diff
@@ -1,155 +1,99 @@
-    "production": [
-      ">0.1%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+    "react-query": "^3.34.6",
+    "seamless-scroll-polyfill": "^2.1.5"
   },
   "devDependencies": {
+    "@babel/core": "^7.16.5",
     "@sentry/cli": "^1.71.0",
-    "@storybook/addon-actions": "^6.1.15",
-    "@storybook/addon-essentials": "^6.1.15",
-    "@storybook/addon-links": "^6.1.21",
-    "@storybook/node-logger": "^6.1.21",
-    "@storybook/preset-create-react-app": "^3.2.0",
-    "@storybook/react": "^6.1.15",
+    "@storybook/addon-actions": "^6.4.9",
+    "@storybook/addon-essentials": "^6.4.9",
+    "@storybook/addon-links": "^6.4.9",
+    "@storybook/builder-webpack5": "^6.4.9",
+    "@storybook/cli": "^6.4.9",
+    "@storybook/manager-webpack5": "^6.4.9",
+    "@storybook/react": "^6.4.9",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/react-hooks": "^7.0.2",
-    "@testing-library/user-event": "^13.5.0",
+    "@testing-library/user-event": "^13.4.1",
     "@types/css-mediaquery": "^0.1.1",
     "@types/geojson": "^7946.0.8",
     "@types/google-protobuf": "^3.15.5",
     "@types/jest": "^26.0.24",
     "@types/node": "^14.18.0",
-    "@types/react": "^17.0.37",
+    "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
     "@types/react-gtm-module": "^2.0.1",
     "@types/react-router-dom": "^5.3.2",
     "@types/react-test-renderer": "^17.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.5.0",
+    "babel-jest": "^27.4.5",
+    "babel-loader": "^8.2.3",
     "cross-env": "^7.0.3",
     "css-mediaquery": "^0.1.2",
+    "eslint": "<8.0.0",
+    "eslint-config-next": "12.0.7",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-storybook": "^0.5.5",
+    "eslint-plugin-unused-imports": "^2.0.0",
+    "i18next-browser-languagedetector": "^6.1.2",
+    "i18next-resources-to-backend": "^1.0.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^27.4.5",
     "jest-junit": "^12.2.0",
     "msw": "^0.35.0",
-    "prettier": "^2.4.1",
-    "react-snap": "^1.23.0",
+    "next-router-mock": "^0.6.1",
+    "prettier": "^2.5.1",
     "react-test-renderer": "^17.0.2",
-    "sentry-testkit": "^3.3.7",
-    "timezone-mock": "^1.2.2"
-  },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{ts,tsx}",
-      "!**/node_modules/**",
-      "!src/proto/**",
-      "!src/stories/**",
-      "!src/**/*.stories.tsx"
-    ],
-    "resetMocks": true
+    "sentry-testkit": "^3.3.4",
+    "timezone-mock": "^1.2.2",
+    "typescript": "~4.5.4"
   },
-  "reactSnap": {
-    "puppeteerArgs": [
-      "--no-sandbox",
-      "--disable-setuid-sandbox",
-      "--disable-dev-shm-usage"
-    ]
+  "resolutions": {
+    "next-i18next": "git+https://github.com/Couchers-org/next-i18next.git#39a50ec9"
```

Comment:
Why is `next-i18next` pinned to a specific commit version?

Reply:
we are waiting for PR to be merged upstream